### ISSUE TITLE
supervisor_twiddler support

### ISSUE THREAD

**Issue by arossert** [mlazarov/supervisord-monitor#18]
Can I use supervisor_twiddler plugin in this awesome dashboard?

If no, it will be nice if you can add support in this plugin to enable adding and removing processes from the servers.

**Comment by mlazarov** [comment]
Hi Daemon22,
currently supervirord-monitor doesn't support supervisor_twiddler. It' doesn't support any additional plugins.

supervisor_twiddler looks very nice addition to supervisor and will be cool to be supported by supervirord-monitord. Unfortunately I don't have time for now to add support in supervirord-monitor. Feel free to make make it happen and share with the community.

**Comment by Daemon22** [comment]
Hi,
Unfortunately I don't have any experience in PHP and I have tried to make the changes but with no success.

I changed this in MY_Controller.php:
$this->{$server}->method('supervisor.'.$method);
to:
$this->{$server}->method($method);

And added this to control.php:
function AddProgram($server, $program){
	$command = array('command'=>'some command');
	$program = array("group_name", "program_name", $command);
	$this->_request($server,'twiddler.addProgramToGroup', $program);
	Redirect('/');
}

But Im always getting "INCORRECT_PARAMETERS" from $this->_request().
Other twiddler functions are working so I think that the problem is in the $command parameter.

According to the twiddler site I need to give a dict with the option.

If you have any idea for a quick solution it will be great.

Thanks allot.
